video/out/vo_gpu_next.c

```diff
@@ -186,6 +188,10 @@ const struct m_opt_choice_alternatives lut_types[] = {
 #define OPT_BASE_STRUCT struct gl_next_opts
 const struct m_sub_options gl_next_conf = {
     .opts = (const struct m_option[]) {
+        {"sub-hdr-peak", OPT_CHOICE(sub_hdr_peak, {"sdr", 0}),
+            M_RANGE(10, 203)},
```

Comment:
Why different ranges? Couldn't we just use `M_RANGE(10, 10000)` for both of them?

Reply:
> Why different ranges? Couldn't we just use `M_RANGE(10, 10000)` for both of them?

As I mentioned in the code:
```c
// The maximum target peak of HDR text subtitle should not exceed the SDR white
```
Text captioning is hardly ever done in HDR space, because the current ass de facto standard is only designed for SDR. 
It is wrong to set the subtitle brightness higher than 203nit for text subtitle production targeting SDR, it will make the text subtitle too bright especially in HDR output.

Reply:
It's on user to set it correctly. We shouldn't impose this limitation in code, because things might change or something and this limitation is artificial. Please use `M_RANGE(10, 10000)` for both of them.

Reply:
Done.

Reply:
After further testing realized that text subtitle brightness above 203 nit is actually useful for inverse mapping.